tests/test_brave_driver.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import pytest
+from selenium import webdriver
+
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.utils import ChromeType, os_name, OSType
+
+
+def test_driver_with_ssl_verify_disabled_can_be_downloaded(ssl_verify_enable):
+    os.environ['WDM_SSL_VERIFY'] = '0'
+    custom_path = os.path.join(
+        os.path.dirname(os.path.dirname(__file__)),
+        "ssl_disabled",
+    )
+    driver_path = ChromeDriverManager(
+        version="83.0.4103.39",
+        path=custom_path,
+        chrome_type=ChromeType.BRAVE,
+    ).install()
+
+    assert os.path.exists(driver_path)
+
+
+def test_brave_manager_with_specific_version():
+    bin_path = ChromeDriverManager("2.27", chrome_type=ChromeType.BRAVE).install()
+    assert os.path.exists(bin_path)
+
+
+def test_brave_manager_with_selenium():
+    binary_location = {
+        OSType.LINUX: "/usr/bin/brave-browser",
+        OSType.MAC: "/Applications/Brave Browser.app/Contents/MacOS/Brave Browser",
+        OSType.WIN: "$env:LOCALAPPDATA\\AppData\\Local\\BraveSoftware\\Brave-Browser\\Application",
```

Comment:
Test fails on Windows CI for now.
Need to fix this path for Github Actions and local installations - make it more universal.

Reply:
Fixed